Source/Eto/Forms/Controls/Splitter.cs

```diff
@@ -87,11 +89,40 @@ protected virtual void OnPositionChanged(EventArgs e)
 
 		#endregion
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:Eto.Forms.Splitter"/> class.
+		/// </summary>
+		public Splitter()
+		{
+			panel1MinimumSize = 0;
+			panel2MinimumSize = 0;
```

Comment:
I know, it's just my preference to set the default values for all the variables in the constructor (so you can easily see what the starting value of something is).